actions/approve-pr/src/main.ts

```diff
@@ -35,3 +35,20 @@ export async function run(): Promise<void> {
     }
   }
 }
+
+function getMergeMethod(): MergeMethod {
```

Comment:
Even if the value was supplied, we would've had the wrong value if someone supplied a lowercase version of it. Here we map to make sure it's the same value. We could do this more succinctly with `getInput('merge_method').toUpperCase()`, but I prefer to explicitly validate the input and return an enum value rather than a string, for type safety.